rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/rules/CreateIssueSummaryDataRuleProvider.java

```diff
@@ -82,22 +82,14 @@ private void generateDataSummary(GraphRewrite event)
                     ObjectMapper objectMapper = new ObjectMapper(jsonFactory);
                     Map<Severity, List<ProblemSummary>> summariesBySeverity =
                         ProblemSummaryService.getProblemSummaries(
-                            event.getGraphContext(), inputApplication, Collections.EMPTY_SET, Collections.EMPTY_SET);
+                            event.getGraphContext(), inputApplication, Collections.<String>emptySet(), Collections.<String>emptySet());
```

Comment:
I'm not sure that it really matters, tbh. I'd be ok with it either way in this case.